specification/databricks/resource-manager/Microsoft.Databricks/preview/2022-10-01-preview/accessconnector.json

```diff
@@ -0,0 +1,430 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "DatabricksAccessConnectorClient",
+    "version": "2022-10-01-preview",
+    "description": "These APIs allow end users to operate on Azure Databricks Access Connector resources."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Databricks/accessConnectors/{connectorName}": {
+      "get": {
+        "tags": [
+          "AccessConnector"
+        ],
+        "operationId": "AccessConnectors_Get",
+        "description": "Gets an azure databricks accessConnector.",
+        "x-ms-examples": {
+          "Get an azure databricks accessConnector": {
+            "$ref": "./examples/AccessConnectorGet.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/ConnectorName"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns the accessConnector.",
+            "schema": {
+              "$ref": "#/definitions/AccessConnector"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../preview/2022-04-01-preview/databricks.json#/definitions/ErrorResponse"
```

Comment:
Could you use the "ErrorDetail" object from common definitions?

Reply:
Unfortunately no there are few differences with that object and the version we have. It is out of scope of current feature.